### PR TITLE
added system diagnostic test checking hb reset

### PR DESCRIPTION
**Pull request by brunofalmeida**

### CODE REVIEW

File: harness_tests/system-diagnostic/main1.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+#include "../../src/general.h"
+
+#define EPS_RESTART_COUNT 10
+#define EPS_RESET_REQUEST 7
+#define EPS_RESET_REASON 11
+
+#define OBC_RESTART_COUNT 8
+#define OBC_RESET_REQUEST 5
+#define OBC_RESET_REASON 9
```
Comment (brunofalmeida) [inline harness_tests/system-diagnostic/main1.c]: You don't need these constants right?

File: harness_tests/system-diagnostic/main1.c [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+/* Resets the OBC SSM and verifies that the reset counter increments correctly */
+void obc_reset_test(void){
+    uint32_t stale_num_restarts = 0x00;
+
+    hb_send_obc_req = true;
+    run_hb();
+    stale_num_restarts = hb_latest_restart_count;
+
+    send_hb_reset(HB_PAY);
```
Comment (brunofalmeida) [inline harness_tests/system-diagnostic/main1.c]: Fix this to be HB_OBC

File: harness_tests/system-diagnostic/main1.c (hunk 1 above)
Comment (nikoog) [reply harness_tests/system-diagnostic/main1.c]: nope, overlooked that earlier- removed now

File: harness_tests/system-diagnostic/main1.c (hunk 2 above)
Comment (nikoog) [reply harness_tests/system-diagnostic/main1.c]: whoops- fixed!

**Comment by brunofalmeida** [comment]
Looks good, you can merge it now

**Pull request merged**